pwa/app_settings.py

```diff
@@ -107,3 +107,17 @@
 ])
 PWA_APP_DIR = getattr(settings, 'PWA_APP_DIR', 'auto')
 PWA_APP_LANG = getattr(settings, 'PWA_APP_LANG', 'en-US')
+PWA_APP_SCREENSHOTS = getattr(settings, 'PWA_APP_SCREENSHOTS', [
+    {
+      'src': '/static/images/icons/splash-750x1334.png',
+      'sizes': '750x1334',
+      "type": "image/png"
+    }
+  ])
+PWA_APP_SHORTCUTS = getattr(settings, "PWA_APP_SHORTCUTS", [
```

Comment:
I'm wondering if we should add the Home as a default shortcut or leave it empty 🤔

Reply:
I think it wouldn't add much, and might be counter intuitive, as shortcuts is an optional field, so if nothing is changed from the default settings, I would expect it to be empty.
 
If a user wants a shortcut to Hom, then they can always add it.